openedx/adg/lms/applications/helpers.py

```diff
@@ -513,6 +519,19 @@ def get_application_hub_instructions(
             congrats = WRITTEN_APPLICATION_COMPLETION_CONGRATS
             message = WRITTEN_APPLICATION_COMPLETION_MSG
 
+    user_application = getattr(user_application_hub.user, 'application', None)
+    if user_application:
+        username = user_application.user.username
```

Comment:
We are using 2 different relations to access the user, How about getting the user once and save it in a variable and use at both places.

```suggestion
    user = user_application_hub.user
    user_application = getattr(user, 'application', None)
    if user_application:
        username = user.username
```